spacy_curated_transformers/tokenization/bbpe_encoder.py

```diff
@@ -49,6 +50,12 @@ def byte_bpe_encoder_forward(
     bos_piece: str = model.attrs["bos_piece"]
     eos_piece: str = model.attrs["eos_piece"]
     unk_piece: str = model.attrs["unk_piece"]
+    initialized: bool = model.attrs["initialized"]
```

Comment:
Does this potentially break bwd-compat, for models that didn't yet have this attribute? If so, should this target 0.3.0?

Reply:
Ack, good catch! Re-targeted.